3languajesTemplate.js

```diff
@@ -0,0 +1,116 @@
+/*
```

Comment:
this needs to be in 3 different files and those files. .css .html .js in the html you'll call in the html the css:
<link rel="stylesheet" href="styles.css">

abd the js <script src="myscripts.js"></script>

then you'll just open the html file which will be called in a browser window